posix.mak

```diff
@@ -130,14 +108,16 @@ $(CURL_TOOLS): $(ROOT)/%: %.d
 $(TOOLS) $(DOC_TOOLS): $(ROOT)/%: %.d
 	$(DMD) $(MODEL_FLAG) $(DFLAGS) -of$(@) $(<)
 
-$(TAGS): %.tag: $(DOC)/%.html $(ROOT)/findtags
-	$(ROOT)/findtags $< > $@
+$(DOC)/chmgen : $(DOC)/chmgen.d
+	$(DMD) -g -of$@ $<
+
+ALL_OF_PHOBOS_DRUNTIME_AND_DLANG_ORG = # ???
```

Comment:
> That won't work without another hand maintained list of all files.

Yeah.

> Rather rebuild it every time.

Is this really the best solution? For such a relatively unimportant utility, I think in makes more sense to let it rebuild only after a `make clean`.